pkg/bot/slack_cloud_test.go

```diff
@@ -81,24 +81,24 @@ func TestWithRetriesFunc(t *testing.T) {
 
 	t.Run("Respected canceled context", func(t *testing.T) {
```

Comment:
the test name should change.  BTW shouldn't we add it just as a new test?

Reply:
Actually I wanted to test it in unit test but it became too hard (as essentially I would need to test the whole `start` that connects to grpc service) so I abandoned the idea but it seems I forgot to revert all changes in this file. Fixed that.